ci/tests/general/general_alba.py

```diff
@@ -334,12 +334,14 @@ def filter_disks(disk_names, amount, disk_type):
         :param disk_type: Type of disk
         :return: Filtered disks
         """
-        grid_ip = General.get_config().get('main', 'grid_ip')
-        storagerouter = GeneralStorageRouter.get_storage_router_by_ip(ip=grid_ip)
-        root_client = SSHClient(storagerouter, username='root')
-        hdds, ssds = GeneralDisk.get_physical_disks(client=root_client)
-        count = 0
-        filtered_disks = list()
+        hdds = dict()
+        ssds = dict()
+        sorted_disks = dict()
+        filtered_disks = dict()
+
+        for node in AlbaNodeList.get_albanodes():
+            root_client = SSHClient(node.ip, username='root')
+            hdds[node.guid], ssds[node.guid] = GeneralDisk.get_physical_disks(client=root_client)
 
         if disk_type == 'SATA':
```

Comment:
If i misspell SATA or SSD or use lowercase words i suddenly get results for all types